tests/union_duplicates.pyi

```diff
@@ -2,17 +2,16 @@
 import builtins
 import typing
 from collections.abc import Mapping
-from typing import (  # Y022 Use "type[MyClass]" instead of "typing.Type[MyClass]" (PEP 585 syntax)
-    Type,
-    Union,
+from typing import (
+    Type,  # Y022 Use "type[MyClass]" instead of "typing.Type[MyClass]" (PEP 585 syntax)
 )
+from typing import Union
 
 import typing_extensions
-from typing_extensions import (  # Y022 Use "type[MyClass]" instead of "typing_extensions.Type[MyClass]" (PEP 585 syntax)
-    Literal,
-    Type as Type_,
-    TypeAlias,
+from typing_extensions import (
+    Type as Type_,)  # Y022 Use "type[MyClass]" instead of "typing_extensions.Type[MyClass]" (PEP 585 syntax
```

Comment:
Looks like an extra parenthesis here, and not sure why you split this into two ImportFroms.

Reply:
i didn't, isort did. It also introduced the syntax error. I'm not impressed.....

Reply:
```suggestion
    Type as Type_,  # isort: skip  # Y022 Use "type[MyClass]" instead of "typing_extensions.Type[MyClass]" (PEP 585 syntax)
```